ARM-wvd-templates/nestedtemplates/managedDisks-customimagevm.json

```diff
@@ -357,31 +357,31 @@
                 }
             }
         },
-        {
-            "apiVersion": "2018-10-01",
-            "type": "Microsoft.Compute/virtualMachines/extensions",
-            "name": "[concat(parameters('rdshPrefix'), add(copyindex(), parameters('vmInitialNumber')), '/', 'dscextension')]",
-            "location": "[parameters('location')]",
-            "dependsOn": ["rdsh-domain-join-loop"],
-            "copy": {
-                "name": "rdsh-dsc-loop",
-                "count": "[parameters('rdshNumberOfInstances')]"
-            },
+      {

```

Comment:
fix indentation

Reply:
A new PR is created, which contains all the latest changes.  Please close this PR.  thanks.